anchore_engine/analyzers/hints.py

```diff
@@ -64,15 +64,19 @@ def __init__(self, pkg):
     def resolve_rpm_fields(self):
         from anchore_engine.util.rpm import split_rpm_filename
 
+        if self.release:
+            rpm = "{}-{}-{}.{}.rpm".format(
```

Comment:
Are these the only two rpm filename formats we could hit?

Reply:
Those should be the only formats. See [RPM File Naming Convention](http://ftp.rpm.org/max-rpm/ch-rpm-file-format.html). The old code assumed the version and release were in one string. Hints allow version and release to be broken out into separate fields which wasn't accounted for.

Reply:
Ack, thanks